aform/src/components/AForm.vue

```diff
@@ -1,26 +1,30 @@
 <template>
 	<form>
 		<component
-			v-for="(componentObj, key) in schema"
+			v-for="(componentObj, key) in modelValue"
 			:is="componentObj.component"
 			:key="key"
 			:schema="componentObj"
+			v-model="childModels[key].value"
 			:data="formData[componentObj.fieldname]"
 			v-bind="componentProps(componentObj)">
 		</component>
 	</form>
 </template>
 
 <script setup lang="ts">
-import { ref } from 'vue'
+import { ref, computed } from 'vue'
 
 import { SchemaTypes } from 'types'
 
 const props = defineProps<{
+	modelValue: SchemaTypes[]
 	schema: SchemaTypes[]
```

Comment:
Should we ask the client to [specify a name for `v-model`](https://v3-migration.vuejs.org/breaking-changes/v-model.html#v-model-arguments)? That would help avoid duplicate props.

It will have to be documented, but instead of `v-model` and `update:modelValue`, the client would have to use `v-model:schema` and `update:schema`, which would be more explicit.

Reply:
Changes to schema from a component don't need to emit. We want it to be reactive so that changes from outside the component are propagated, but there's no change to a component's schema that should bubble back up.